subcommand/control-plane/command_test.go

```diff
@@ -439,6 +447,10 @@ func TestRun(t *testing.T) {
 				},
 				Partition: expectedPartition,
 				Namespace: expectedNamespace,
+				ServiceLocality: &api.Locality{
+					Region: testRegion,
+					Zone:   testZone,
+				},
```

Comment:
non-blocking: Do you think it's worth testing the case where we return a `nil` locality? Given that the `AWS_REGION` will always be provided by the ECS environment this would really just be a code coverage case.